pkg/minikube/driver/driver.go

```diff
@@ -212,3 +213,12 @@ func SetLibvirtURI(v string) {
 	os.Setenv("LIBVIRT_DEFAULT_URI", v)
 
 }
+
+//MachineName returns the name of the machine given the cluster and node names
```

Comment:
Add a space after "//".

Also, it may be worth clarifying here that driver.MachineName returns the machine name here is "as seen by the hypervisor". 

I would think other users who don't care about the hypervisor view would use Node.Name directly.